ui/admin/setup-edit.php

```diff
@@ -257,13 +257,13 @@
                     <span>&nbsp;</span>
                 </th>
                 <th scope="col" id="label" class="manage-column field-label">
-                    <span>Label<?php pods_help( __( "<h6>Label</h6>The label is the descriptive name to identify the Pod field.", 'pods' ) ); ?></span>
+                    <span><?php _e( 'Label', 'pods' ); pods_help( "<h6>" . __( 'Label', 'pods' ) . "</h6>" . __( 'The label is the descriptive name to identify the Pod field.', 'pods' ) ); ?></span>
```

Comment:
Please use `esc_html_e()` instead of `_e()` for echoing values that do not require HTML.